torch/optim/adadelta.py

```diff
@@ -5,9 +5,32 @@
 
 
 class Adadelta(Optimizer):
-    """Implements Adadelta algorithm.
-
-    It has been proposed in `ADADELTA: An Adaptive Learning Rate Method`__.
+    r"""Implements Adadelta algorithm.
+
+    .. math::
+       \begin{aligned}
+            &\rule{110mm}{0.4pt}                                                                 \\
+            &\textbf{input}      : \gamma \text{ (lr)}, \: \theta_0 \text{ (params)},
+                \: f(\theta) \text{ (objective)}, \: \rho \text{ (decay)}, \: weightdecay        \\
+            &\textbf{initialize} :  E[g^2]_0  \leftarrow 0, \: E[\Delta x^2]_0  \leftarrow 0
```

Comment:
Why not name these like the other optimizers (like exp_avg) I feel like this would be easier to read?

Reply:
sounds good ! and done